platform/src/main/java/org/stellar/anchor/platform/callback/PlatformIntegrationHelper.java

```diff
@@ -50,34 +50,31 @@ public static AnchorException httpError(String responseContent, int responseCode
         "Error returned from the Anchor Backend.\nresponseCode={}\nContent={}",
         responseCode,
         responseContent);
-    ErrorResponse errorResponse;
+    ErrorResponse errorResponse = null;
```

Comment:
This is not necessary.

Reply:
It's actually necessary to initialize it because it's a local variable. It wouldn't be required for a class/instance variable though. Here's a blog post on the subject: https://www.baeldung.com/java-error-variable-initialized

BTW, I only know it's needed because Intellij told me 😅